api/v1alpha1/virtualmachineimage_conversion.go

```diff
@@ -308,6 +345,11 @@ func (src *VirtualMachineImage) ConvertTo(dstRaw conversion.Hub) error {
 		return err
 	}
 
+	if err := convert_v1alpha1_VirtualMachineImageAnnotations_To_v1alpha2_VirtualMachineImage_VMwareSystemProperties(
```

Comment:
We want to remove the annotations that we're moved to the status. One of the goals w/ this is to make MD-only watches more effective by not have the big TKG yaml blog in the ObjectMeta.

Reply:
fixed it.. PTAL when you get a chance